### ISSUE TITLE
Consolidate rdev explorer+portal deployment

### ISSUE THREAD

**Issue by ebezzi** [chanzuckerberg/single-cell-data-portal#2422]
TODOs:

- [x] Remove happy code from single-cell-infra
- [x] Move the relevant happy code from single-cell-infra to single-cell-explorer
- [ ] Create a GHA workflow that does rdev deployment for both jobs (@elopez will do this)
- [ ] Test the system
- [ ] Update docs [here](https://docs.google.com/document/d/19IZbojtc7eofV75NnL5C6fCgBJB13KIswWNOn12xAa0/edit#heading=h.2enfpkkmdq6f) and [here](https://docs.google.com/document/d/1nynGcBS_TA55qlQo9WjINGkcMnE_xIBz7-inmop2bqo/edit#heading=h.mk7hihj64gw4)

**Comment by maniarathi** [comment]
Blocked by a need for the Happy team to deploy two repos to the same rDev environment.

**Comment by atolopko-czi** [comment]
From last week's engineering meeting, there was some appetite from the engineers to get this addressed, since it would help simplify their pre-merge testing processes. Next step is to contact Happy team to ensure the necessary work is being tracked by them and then link this issue to their work.

**Comment by ebezzi** [comment]
I can keep working on this if the team thinks it's useful. It will require a few weeks as the Happy team will probably have to make a few changes to make this work.

**Comment by metakuni** [comment]
Discussed with Emanuele and closing this since there is currently no need for consolidated deployment. A new issue will be created when/if the need arises.

**Comment by atolopko-czi** [comment]
> Discussed with Emanuele and closing this since there is currently no need for consolidated deployment. A new issue will be created when/if the need arises.

Can you elaborate on "no need for consolidated deployment"? In light of our CI/CD plans, won't we need to be able to test Explorer<>Portal interactions in RDev? Are we just reasoning that we haven't need to test this _recently_?

**Comment by ebezzi** [comment]
Given how slow the explorer deployment is, I don't think we can justify doing this now. Happy/ECS didn't introduce any new tool that would help with this, so this is heavy lift work still. It's possible that Happy/EKS could simplify the work, but since we didn't adopt it, I don't think this can be scheduled anytime soon.

**Comment by atolopko-czi** [comment]
Thanks! I suppose we'll have to (re-)roll our own solution for Explorer<>Portal rdev testing when needed. Most importantly, each stack will continue to independently deployed, which is probably the right level of control for testing needs.
